Gruntfile.js

```diff
@@ -43,16 +43,31 @@ module.exports = function (grunt) {
 		sauceLoginFile: grunt.file.exists('SAUCE_API_KEY.yml') ? grunt.file.readYAML('SAUCE_API_KEY.yml') : undefined,
 		sauceUser: process.env.SAUCE_USERNAME || 'fuelux',
 		sauceKey: process.env.SAUCE_ACCESS_KEY ? process.env.SAUCE_ACCESS_KEY : '<%= sauceLoginFile.key %>',
-		allTestUrls: ['2.1.0', '1.11.0', '1.9.1', 'browserGlobals'].map(function (ver) {
-			if (ver === 'browserGlobals') {
-				return 'http://localhost:<%= connect.testServer.options.port %>/test/fuelux-browser-globals.html';
+		// TEST URLS
+		distTestUrl: ['http://localhost:<%= connect.testServer.options.port %>/test/?testdist=true'],
+		allTestUrls: ['2.1.0', '1.11.0', '1.9.1', 'browserGlobals', 'codeCoverage' ].map(function (type) {
+			if (type === 'browserGlobals') {
+				return 'http://localhost:<%= connect.testServer.options.port %>/test/browser-globals.html';
+			}
+			else if (type === 'codeCoverage') {
+				return 'http://localhost:<%= connect.testServer.options.port %>/test/?coverage=true';
+			}
+			else {
+				// test dist with multiple jQuery versions 
+				return '<% distTestUrl %>'; //'http://localhost:<%= connect.testServer.options.port %>/test/?jquery=' + ver + '&testdist=true';
 			}
-
-			return 'http://localhost:<%= connect.testServer.options.port %>/test/fuelux.html?jquery=' + ver + '&testdist=true';
 		}),
-		testUrl: ['http://localhost:<%= connect.testServer.options.port %>/test/fuelux.html?jquery=' + '1.9.1&testdist=true'],
 
 		//Tasks configuration
+		blanket_qunit: {
+			source: {
+				options: {
+					urls: ['http://localhost:<%= connect.testServer.options.port %>/test/?coverage=true&gruntReport'],
+					threshold: 68,
+					globalThreshold: 81
```

Comment:
So, what _exactly_ happens if this fails? Is there ever a possibility of it failing during a servefast or a normal serve, and blocking development? I understand with TDD, you write your tests first, but, we don't actually do that here to my knowledge. I'm not really ok with this being anything other than an informational tool. If it throws a warning or something that says "Your PR is likely to be rejected because it appears to not have proper unit tests" that's really neat and useful imo, but, failing out of the grunt task entirely is... unsavory to me.